examples/tanstack-router/README.md

```diff
@@ -0,0 +1,13 @@
+# A minimal React tRPC example using @tanstack/react-router
```

Comment:
Just a playground for seeing the usage - I don't think we should keep this but instead contribute to https://github.com/TanStack/router/tree/beta/examples/react/with-trpc-react-query

Reply:
Seems like they haven't solved the data loading either? https://github.com/TanStack/router/blob/484af7fd844c3d4b02d67846ec268bc7310b9aba/examples/react/with-trpc-react-query/client/main.tsx#L111-L114

Reply:
Maybe just remove the example for now then and revisit later?